.github/workflows/convert-branch-name.yml

```diff
@@ -5,24 +5,59 @@ on:
   workflow_call:
     inputs:
       branch:
-        description: "original branch name"
+        description: "Original branch name"
         required: true
         type: string
     outputs:
       namespace_from_branch:
-        description: "branch in namespace compatible fashion"
+        description: "Branch in namespace compatible fashion"
         value: ${{ jobs.convert_branch_name.outputs.namespace_from_branch }}
+      image_tag_from_branch:
+        description: "Image tag from branch"
+        value: ${{ jobs.convert_branch_name.outputs.image_tag }}
+      chart_version_from_branch:
+        description: "Chart version from branch"
+        value: ${{ jobs.convert_branch_name.outputs.chart_version }}
 
 jobs:
   convert_branch_name:
     runs-on: ubuntu-latest
     outputs:
-      namespace_from_branch: ${{ steps.create_branch_identifier.outputs.namespace_from_branch }}
+      namespace_from_branch: ${{ steps.create_namespace.outputs.namespace_from_branch }}
+      image_tag: ${{ steps.create_image_tag.outputs.image_tag }}
+      chart_version: ${{ steps.create_chart_version.outputs.chart_version }}
     # This step will match and return either text-1234 or text-1234-1234 from the input branch name
     steps:
-      - id: create_branch_identifier
+      - id: create_namespace
```

Comment:
i think this name is a bit misleading, a name is generated for the namespace, not a namespace

Reply:
going with identifier